apps/web/src/server/app/lpa/lpa.pipes.js

```diff
@@ -0,0 +1,69 @@
+import { body } from 'express-validator';
+
+// All validation pipes will save into the current request all the validation errors that would be used
+// by the `expressValidationErrorsInterceptor` to populate the body with.
+
+/**
+ * Validate the questionnaire form to ensure any items marked as missing or incorrect have details provided, if there is a details field provided
+ *
+ * @returns {void}
+ */
+export const validateQuestionnairePipe = () => [
```

Comment:
I realise this has been done elsewhere already, but I am not really sure why we are making these pipes into functions, only to call them without arguments. Leave as is I guess, just voicing curiosity!

Reply:
It's a good point! We can discuss with Radu when he's back

Reply:
@radum another one for you to be aware of

Reply:
I think I just copy pasted the example from the docs but true there is no reason for these to be functions, so you can change them here.

Reply:
Updated!